Do not fail if related usns fail (SC-1300) (#2497)
canonical/ubuntu-pro-client: docs/tutorials/fix_scenarios.md

that we know has no fix available by running:

```console
$ sudo apt-get install -y expat=2.1.0-7 swish-e matanza ghostscript
```

Now, we can confirm that there is no fix by running the following command:

```console
$ pro fix CVE-2017-9233
```

You will see the following output:

```
CVE-2017-9233: Coin3D vulnerability
  - https://ubuntu.com/security/CVE-2017-9233

3 affected source packages are installed: expat, matanza, swish-e
(1/3, 2/3) matanza, swish-e:
Ubuntu security engineers are investigating this issue.
(3/3) expat:
A fix is available in Ubuntu standard updates.
{ apt update && apt install --only-upgrade -y expat }

2 packages are still affected: matanza, swish-e
✘ CVE-2017-9233 is not resolved.
```

> As you can see, we are informed by `pro fix` that some package do not have a fix available. In

Reviewer: ```suggestion
As you can see, we are informed by `pro fix` that some packages do not have a fix available. In
```

Technical support level: essential
{ apt update && apt install --only-upgrade -y curl libcurl3-gnutls }
✔ USN-5079-2 is resolved.

Found related USNs:
- USN-5079-1

Fixing related USNs:
- USN-5079-1
No affected source packages are installed.

✔ USN-5079-1 does not affect your system.

Summary:
✔ USN-5079-2 [requested] is resolved.
✔ USN-5079-1 [related] does not affect your system.
```

Note that we can see that this command also fixed related USNs **USN-5079-1**.

> If you want to learn more about related USNs, please that a look into this [document](../references/what_are_related_usns)

Reviewer: ```suggestion
If you want to learn more about related USNs, refer to [our explanation guide](../references/what_are_related_usns).
```

Reviewer: This link may change if we combine all three pages - we can point to the header so it jumps to the right part of the page.
